src/main/java/com/khutircraftubackend/product/ProductMapper.java

```diff
@@ -44,6 +44,9 @@ default void validateCreateRequest(ProductCreateRequest request) {
         if (request.seller() == null) {
             throw new IllegalArgumentException(ProductValidationMessages.SELLER_NULL);
         }
+        if(request.categoryId() == null) {
+            throw new IllegalArgumentException(ProductValidationMessages.CATEGORY_NULL_OR_EMPTY);
+        }
```

Comment:
a minor ask really, 
could you please add empty lines between each `if ()` block? 
that would improve readability a bit :)

also, try to be consistent everywhere with your style:
```java
    if (some.condition.here()) {
   // ^ space              // ^ space
    }
```

Reply:
I will try.